static/compiler.js

```diff
@@ -361,6 +361,12 @@ Compiler.prototype.getEffectiveFilters = function () {
     if (filters.execute && !this.compiler.supportsExecute) {
         delete filters.execute;
     }
+    var disabledFilters = this.compiler.disabledFilters;
+    disabledFilters.forEach(function (filter) {
+        if (filters[filter] && disabledFilters.includes(filter)) {
```

Comment:
As above, use `.indexOf > 0` instead of includes on the client side code (You're free to use it on the server side code tho!)